packages/keyValueCache/src/InMemoryLRUCache.ts

```diff
@@ -32,12 +39,13 @@ export class InMemoryLRUCache<V extends {} = string>
     return 1;
   }
 
-  async set(key: string, value: V, options?: KeyValueCacheSetOptions) {
-    if (options?.ttl) {
-      this.cache.set(key, value, { ttl: options.ttl * 1000 });
-    } else {
-      this.cache.set(key, value);
-    }
+  async set(key: string, value: V, options?: SO) {
+    // If a TTL in seconds is provided, convert it to milliseconds.
+    // Otherwise, default it to 0 to indicate "no TTL".
+    this.cache.set(key, value, {
+      ...options,
+      ttl: options?.ttl ? options.ttl * 1000 : 0,
```

Comment:
I'm not sure `0` is the right thing to do here.

[The docs say if it's set at all, it needs to be a positive number](https://github.com/isaacs/node-lru-cache#ttl). So if this works, it's undocumented behavior. I would prefer this to remain omitted if not set.